promitor-agent-scraper/values.yaml

```diff
@@ -175,9 +177,28 @@ rbac:
     name: promitor-scraper
     annotations: {}
 
+    ## Set this to true if you plan on using Pod Security Policy
+    automountServiceAccountToken: false
+
 nodeSelector: {}
 
+## securityContext and containerSecurityContext are using secure defaults.
+## Only override if you have a good reason to.
 securityContext:
-  enabled: false
+  enabled: true
+  runAsGroup: 10000
+  runAsNonRoot: true
+  runAsUser: 10000
+  seccompProfile:
+    type: RuntimeDefault
+
+containerSecurityContext:
```

Comment:
That's OK for me, but would you mind opening an issue to improve this for our next major version please?

Reply:
I can do that, but is that really necessary in this case? Breaking changes are breaking for a reason and it only makes sense to do so if the benefits outweigh the annoyance users have to deal with. In this case we have some minor terminology difference, it does not really matter if properties are called `securityContext` and `containerSecurityContext` or `podSecurityContext` and `securityContext` as long as they template correct yaml places and it's clear to the user what is going on.
As I said, I've seen usage of both in the wild, and I'd just leave it like this, rather than make a breaking change for "correctness" sake.

Reply:
It's a matter of being consistent but I'll see later down the road.